osm2gtfs/core/osm_connector.py

```diff
@@ -618,4 +756,4 @@ def _find_best_name_for_unnamed_stop(self, stop):
                 winner_distance = distance
 
         # take name from winner
-        stop.name = winner.tags["name"].encode('utf-8')
```

Comment:
Because I went over the whole thing and tried to apply the principle of decoding when information is read into Python's Unicode and encoding when it is printed out.